src/models/generated/Bus.jl

```diff
@@ -0,0 +1,61 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+"""A power-system bus."""
+mutable struct Bus <: Injection
+    number::Int64  # number associated with the bus
+    name::String  # the name of the bus
+    bustype::Union{Nothing, BusType}  # bus type
+    angle::Union{Nothing, Float64}  # angle of the bus in degrees
```

Comment:
the angle of the bus in degrees in radians, this was a mistake from old code.

Reply:
Just to be crystal clear:  are you saying it should it be "angle of the bus in radians"?

Reply:
yes